docs/guides/compute-freeform.mdx

```diff
@@ -42,6 +42,32 @@ origin|destinations|percentile|cutoff|access
 2|Total Jobs|5|30|142600
 etc
 
+## Dual Accessibility
+The time needed to reach a threshold number of destinations (such as a health center or three grocery stores), also known as ("dual" accessibility)[https://doi.org/10.1111/gean.12220], can be calculated with an experimental feature in Conveyal.
+
+This feature is not yet available through the regional analysis creation interface. To activate it, add the following to your (custom analysis request JSON)[analysis/configuration#additional-settings].
+
+```JSON
+"includeTemporalDensity": true,
+"dualAccessibilityThreshold": n
+```
+where n is the desired threshold number of opportunities.
+
+The resulting CSV file will have the following columns:
+* origin: Id of the origin point
+* destinations: Id of the destination pointset
+* percentile: travel time percentile
+* D: the number of minutes required to reach a threshold of n opportunities. A value of -1 indicates the specified threshold is not reached within the maximum cutoff specified when the regional analysis was created.
```

Comment:
```suggestion
* dual: the number of minutes required to reach a threshold of n opportunities. A value of -1 indicates the specified threshold is not reached within the maximum cutoff specified when the regional analysis was created.
```

Reply:
Oops, I made this change locally but didn't save before comitting.